content/en/agent/fleet_automation/setup.md

```diff
@@ -0,0 +1,144 @@
+1. Restart your Agent.
+
+### Agent Remote Configuration status
+
+Gain visibility into the Remote Configuration status of your Agent through the [Remote Configuration UI][3].
+
+The following table describes the meaning of each Agent status:
+
+  | Agent Status     | Description                                      |
+  |------------------|--------------------------------------------------|
+  | CONNECTED      | The Agent deployed in your environment is able to reach, authenticate, and authorize successfully to Datadog. This is the optimal state you want your Agents to be in for Remote Configuration.                                               |
+  | UNAUTHORIZED          | The Agent deployed in your environment is able to reach Datadog but is not able to authenticate and authorize with Datadog for Remote Configuration operation. The most likely cause is the API Key used by the Agent is not Remote Configuration-enabled. To fix the issue, enable Remote Configuration capability on the API Key used by the Agent.                                                 |
+  | CONNECTION ERROR        |   The Agent deployed in your environment has `remote_config.enabled` set to true in its `datadog.yaml` configuration file, however, the Agent cannot be found in the Remote Configuration service. The most likely cause is that the Agent is unable to reach Remote Configuration [endpoints][4]. To fix the issue, allow outbound HTTPS access to Remote Configuration endpoints from your environment. This status displays when the Agent version is `7.45.0` or higher.
+  | DISABLED       |   The Agent deployed in your environment has `remote_config.enabled` set to false in its `datadog.yaml` configuration file. Set `remote_config.enabled` to true if you want to enable Remote Configuration on the Agent. This status displays when the Agent version is `7.45.0` or higher. |
+  | NOT CONNECTED       | The Agent cannot be found in the Remote Configuration service and could have `remote_config.enabled` set to true or false in its `datadog.yaml` configuration file. Check your local Agent configuration or your proxy settings. This status displays when the Agent version is higher than `7.41.1` but lower than `7.45.0`.            |
+  | UNSUPPORTED AGENT   | The Agent is on a version that is not Remote Configuration capable. To fix this issue, update the Agent to the latest available version. |
+
+### Opting out of Remote Configuration
+
+To opt out of Remote Configuration use, you can disable Remote Configuration at the organization level. Optionally, you can also disable Remote Configuration capability at the API key level and Agent level.
+
+#### At the Organization level
+
+Disable Remote Configuration at the organization level on the [Remote Configuration][3] page. This disables Datadog components across your organization to receive configurations from Datadog. You need the [`org_management`][1] permission to disable Remote Configuration at the organization level.
```

Comment:
Should we say that doing this will also disable other Remote Config enabled products and link to the list?

Reply:
Yeah, I'd forgotten this paragraph was in here. We're trying to dissuade users from disabling globally, so I'll probably just remove this